vspace/src/main/java/edu/asu/diging/vspace/core/util/CitationFormatter.java

```diff
@@ -0,0 +1,218 @@
+package edu.asu.diging.vspace.core.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CitationFormatter {
```

Comment:
I'm confused about what this class is for. We don't have in text citations, just a list of references, so we don't need any in-text citation formatting.